client/functions.lua

```diff
@@ -43,16 +43,13 @@ end
 function QBCore.Functions.LookAtEntity(entity, timeout, speed)
     local involved = GetInvokingResource()
     if not DoesEntityExist(entity) then
-        turnPromise:reject(involved .. ' :^1  Entity does not exist')
```

Comment:
why removing the reject?

Reply:
`turnPromise` was removed due to the fact that its both undefined and unneeded in this situation. So rejecting here would not work, and just cause an error from trying to index something `nil`.

I guess we should actually `error` or `warn` and return nil instead of returning the error msg

Reply:
qb-core in its current state:
![image](https://github.com/user-attachments/assets/46a84c9b-4c6e-4810-b19a-6e597a7e7e7d)
No definition of `turnPromise`.